geometry_msgs/Quaternion orientation
---
sensor_msgs/Image image